src/main/java/gregtech/common/blocks/BlockCompressed.java

```diff
@@ -157,18 +153,18 @@ public SoundType getSoundType(IBlockState state, @Nonnull World world, @Nonnull
     @Override
     @SideOnly(Side.CLIENT)
     public void onTextureStitch(TextureStitchEvent.Pre event) {
-        for (IBlockState state : this.getBlockState().getValidStates()) {
-            Material m = state.getValue(variantProperty);
-            event.getMap().registerSprite(MaterialIconType.block.getBlockTexturePath(m.getMaterialIconSet()));
-        }
```

Comment:
Is this method intentionally left empty?

Reply:
It still needs declaration because it's abstract.